tabbycat/motions/utils.py

```diff
@@ -0,0 +1,30 @@
+from django.core.exceptions import ValidationError
+from django.db.models import Count
+from django.utils.translation import gettext as _
+
+from .models import DebateTeamMotionPreference
+
+
+def merge_motions(new_bs, bses):
+    n_motions = bses.aggregate(n_motions=Count('motion', distinct=True))['n_motions']
+    if n_motions > 1:
+        raise ValidationError(_("Not all latest ballots list the same motion, so could not be merged."))
+    elif n_motions == 1:
+        new_bs.motion = bses[0].motion
+
+
+def merge_motion_vetos(new_bs, bses):
+    vetos = {}
+    pref_lists = [list(bs.debateteammotionpreference_set.all().values_list(
+        'debate_team', 'debate_team__side', 'motion', 'preference')) for bs in bses]
+    preferences = {p for p in pref_lists}
```

Comment:
```
Traceback (most recent call last):
[...]
  File "/home/czlee/tabbycat/tabbycat/motions/utils.py", line 20, in merge_motion_vetos
    preferences = {p for p in pref_lists}
  File "/home/czlee/tabbycat/tabbycat/motions/utils.py", line 20, in <setcomp>
    preferences = {p for p in pref_lists}

Exception Type: TypeError at /australs24team/admin/results/debate/369/merge/latest/
Exception Value: unhashable type: 'list'
```

I tried changing `list` to `tuple` in line 18, but then I get:
```
  File "/home/czlee/tabbycat/tabbycat/motions/utils.py", line 27, in merge_motion_vetos
    for dt, side, motion, preference in preferences:

Exception Type: ValueError at /australs24team/admin/results/debate/369/merge/latest/
Exception Value: not enough values to unpack (expected 4, got 2)
```

I think that's in turn because it's trying to iterate through `preferences` as if it were a list of 4-tuples, when it's actually a list of 2-tuples of 4-tuples?